sylvia-derive/src/message.rs

```diff
@@ -421,9 +424,9 @@ impl<'a> ContractEnumMessage<'a> {
         let variants_constructors = variants.iter().map(MsgVariant::emit_variants_constructors);
         let variants = variants.iter().map(MsgVariant::emit);
 
-        let ctx_type = msg_ty.emit_ctx_type(&custom.query());
+        let ctx_type = msg_ty.emit_ctx_type(&custom.query_or_default());
```

Comment:
Same here, thinking of `query_or_empty` - an idea, not sure about that. Yes, I know I suggested it before.